receiver/redisreceiver/config.md

```diff
@@ -0,0 +1,270 @@
+| transport |string| tcp | Transport to use. Known protocols are "tcp", "tcp4" (IPv4-only), "tcp6" (IPv6-only), "udp", "udp4" (IPv4-only), "udp6" (IPv6-only), "ip", "ip4" (IPv4-only), "ip6" (IPv6-only), "unix", "unixgram" and "unixpacket".  |
+| password |string| <no value> | Optional password. Must match the password specified in the requirepass server configuration option.  |
+| tls |[tls-TLSClientSetting](#tls-TLSClientSetting)| <no value> | TLSClientSetting contains TLS configurations that are specific to client connections in addition to the common configurations. This should be used by components configuring TLS client connections.  |
+| metrics |[metrics-MetricsSettings](#metrics-MetricsSettings)| <no value> | MetricsSettings provides settings for redisreceiver metrics.  |
+
+### tls-TLSClientSetting
+
+| Name | Field Info | Default | Docs |
+| ---- | --------- | ------- | ---- |
+| ca_file |string| <no value> | Path to the CA cert. For a client this verifies the server certificate. For a server this verifies client certificates. If empty uses system root CA. (optional)  |
+| cert_file |string| <no value> | Path to the TLS cert to use for TLS required connections. (optional)  |
+| key_file |string| <no value> | Path to the TLS key to use for TLS required connections. (optional)  |
+| min_version |string| <no value> | MinVersion sets the minimum TLS version that is acceptable. If not set, refer to crypto/tls for defaults. (optional)  |
+| max_version |string| <no value> | MaxVersion sets the maximum TLS version that is acceptable. If not set, refer to crypto/tls for defaults. (optional)  |
+| reload_interval |[time-Duration](#time-Duration)| <no value> | ReloadInterval specifies the duration after which the certificate will be reloaded If not set, it will never be reloaded (optional)  |
+| insecure |bool| true | In gRPC when set to true, this is used to disable the client transport security. See https://godoc.org/google.golang.org/grpc#WithInsecure. In HTTP, this disables verifying the server's certificate chain and host name (InsecureSkipVerify in the tls Config). Please refer to https://godoc.org/crypto/tls#Config for more information. (optional, default false)  |
+| insecure_skip_verify |bool| <no value> | InsecureSkipVerify will enable TLS but not verify the certificate.  |
+| server_name_override |string| <no value> | ServerName requested by client for virtual hosting. This sets the ServerName in the TLSConfig. Please refer to https://godoc.org/crypto/tls#Config for more information. (optional)  |
+
+### metrics-MetricsSettings
+
+| Name | Field Info | Default | Docs |
+| ---- | --------- | ------- | ---- |
+| redis.clients.blocked |[redis-clients-blocked-MetricSettings](#redis-clients-blocked-MetricSettings)| <no value> | MetricSettings provides common settings for a particular metric.  |
```

Comment:
Same Q as @mx-psi asked for other fields. All the metrics are enabled or disabled by default. Is it possible to get the default value instead of `<no value>`?

Reply:
I understand now that this is a default for `MetricSettings`, not bool, and I see why we need to have an instance per every entry `MetricSettings` in Field Info. 

I think we can do something with this to make it more readable later. 

For now I believe we just need to correctly display zero values, e.g. `false` for bool and `""` for strings